Queue/Backend/Redis.php

```diff
@@ -101,7 +101,7 @@ public function appendValuesToList($key, $values)
         $this->connectIfNeeded();
 
         foreach ($values as $value) {
-            $this->redis->rPush($key, $value);
+            $this->redis->rPush($key, gzdeflate($value));
```

Comment:
We are using `gzcompress` and `gzuncompress` in Matomo for these things, the MySQL backend in Queued tracking uses it as well

Reply:
The reason we want to compress data before sending it to Redis, is to reduce latency and memory requirements in Redis.

This compression only has an affect when QueuedTracking is enabled for Redis, nothing else.

Reply:
Yes. It's just that we use `gzcompress ` everywhere in Matomo instead of `gzdeflate ` so be nice to keep it consistent. If compress is an issue we could potentially use deflate as well maybe.